src/components/calculator-display.js

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import calculate from '../logic/calculate';
+
+const CalculatorDisplay = () => {
+  const [value, setValue] = useState({
+    total: '0',
+    next: null,
+    operation: null,
+  });
+
+  const { total, next, operation } = value;
+
+  const handleClick = (num) => {
+    setValue(calculate(value, num));
+  };
+
+  return (
+    <section>
+      <table>
+        <tr>
+          <th colSpan="4">
+            {total}
+            {operation}
+            {next}
+          </th>
+        </tr>
+        <tr>
+          <td><button type="button" onClick={() => handleClick('AC')}>AC</button></td>
```

Comment:
- [ ] [OPTIONAL] Good job @seyioyemade , but I noticed a little confusion, because  the onclick event is in the button which is inside the td, sometimes when I click the td which is assumed to be the button from your design, nothing happens, this is because of the padding you gave to td in your CSS and since the onlcick method is on the button, I might be clicking outside the button without knowing, so you can make the button width and height 100% and remove the padding in td and put it in button, this should apply for all td and button in this component